tests/browser/heda_settings_accounts_and_contacts_edit_mode.robot

```diff
@@ -0,0 +1,64 @@
+*** Settings ***
+
+Resource        tests/HEDA.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Go To Heda Home
+    Go To Heda Home
```

Comment:
Line 10 here is what's causing the test failure.  Line 9 defines the keyword Go To Heda Home and then inside the definition of the keyword you're calling the keyword being defined.  Deleting line 10 should fix it.